src/process.rs

```diff
@@ -59,6 +59,7 @@ pub struct LaunchOptions {
     pub headless: bool,
     pub port: Option<u16>,
     pub path: std::path::PathBuf,
+    pub load_extension: Option<String>,
```

Comment:
Isn't it possible to load multiple extensions? Then this would be a `Vec<String>`

Reply:
Yes, you can load multiple extensions. you need to pass comma separated string of extensions path.

Reply:
Could you put that in a docstring?

Reply:
Sure, will do it. :)

Reply:
I reckon having this field be a `Vec<String>` and then doing the comma separation (via [`.join()`](https://doc.rust-lang.org/std/slice/trait.SliceConcatExt.html#tymethod.join)) would be pretty neat. And something like `extension_paths` might be a clearer name? But I also don't want to overload you with changes, because what you've already got is good too :)

Reply:
If the CLI-parameter takes a comma-separated string, we should do too IMHO. There could be a method `add_extension`, which pushes a comma and the given name.

Reply:
Fair enough, I'm cool with that. Looks like Puppeteer itself just lets you pass along arbitrary args to the Chrome binary: https://pptr.dev/#?product=Puppeteer&version=v1.12.2&show=api-working-with-chrome-extensions.

On a side note, it's interesting / worrying that loading extensions apparently won't work in headless mode (found via link above). I wonder how many things there are like that?